packages/itwinui-css/src/bridge.scss

```diff
@@ -328,14 +331,24 @@
 
   .iui-input:where(:not([role='combobox'])) {
     // StrataKit input background colors are different from global values
-    --_iui-field-color-background: var(--stratakit-color-bg-control-textbox);
-    --_iui-field-color-background-hover: var(--_iui-field-color-background);
-    --_iui-field-color-background-disabled: var(--_iui-field-color-background);
+    --_iui-field-color-background-disabled: var(--stratakit-color-bg-control-textbox);
   }
 
   .iui-input-flex-container {
     // StrataKit input background colors are different from global values
-    background-color: var(--stratakit-color-bg-control-textbox);
+    --_iui-input-background-color: var(--stratakit-color-bg-neutral-base);
+
+    &:hover {
+      --_iui-input-background-color: color-mix(
+        in oklch,
+        var(--stratakit-color-bg-neutral-base) 100%,
+        var(--stratakit-color-glow-hue) var(--stratakit-color-bg-glow-base-hover-\%)
+      );
+    }
+
+    &[data-iui-disabled='true'] {
+      --_iui-input-background-color: var(--iui-color-background-disabled);
+    }
   }
```

Comment:
This looks wrong? Inputs should be based on `var(--stratakit-color-bg-control-textbox)`

Reply:
I'm seeing `var(--stratakit-color-bg-page-base)` being used on inputs: https://github.com/iTwin/design-system/blob/f1be764489bf7ad912a5bfbfe272f1a7769fc562/packages/bricks/src/TextBox.css#L6.

Created a PR to update StrataKit to use `var(--stratakit-color-bg-control-textbox)` in https://github.com/iTwin/design-system/pull/982.

Fix in https://github.com/iTwin/iTwinUI/pull/2598/commits/dc7658a9cc6126860c3ccb8806f17261b7e3fc1c.